tests/org.pgcase.xobot.dbproc.jdbc.tests/src/org/pgcase/xobot/dbproc/jdbc/tests/SimpleJdbcFunctionTest.java

```diff
@@ -0,0 +1,39 @@
+package org.pgcase.xobot.dbproc.jdbc.tests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.pgcase.xobot.dbproc.jdbc.DbprocJdbc;
+import org.pgcase.xobot.dbproc.jdbc.functions.JdbcFunctionExtractor;
+import org.pgcase.xobot.dbproc.runtime.functions.XFunctionDescriptor;
+
+class SimpleJdbcFunctionTest {
+	
+	private static final String CREATE_TEST_FUNC = "sql/create_test_function1.sql"; //$NON-NLS-1$
+	private static SystemJdbcConnection conn; 
+	
+	@Test
+	void testFunction() throws IOException, SQLException {
+		conn = new SystemJdbcConnection();
```

Comment:
Now the connection created here and closed somewhere in implementation
Let's try to implement more clear connection state management like
1) create
2) pass somebody to use
3) close